crates/fiber-lib/src/fiber/network.rs

```diff
@@ -2966,10 +2981,11 @@ where
         let store = self.store.clone();
         self.peer_session_map
             .insert(remote_peer_id.clone(), (session.id, session.ty));
-        if self
-            .state_to_be_persisted
-            .save_peer_pubkey(remote_peer_id.clone(), remote_pubkey)
```

Comment:
@quake is there any specific reason we didn't store `session.address` here for remote peers?